database/database.js

```diff
@@ -104,6 +105,74 @@ module.exports.addUser = user => {
 
 })
 }
+module.exports.addFavorite = (username, photoObject) => {
+  console.log('INSIDE ADD FAVORITE DATABASE', username, typeof photoObject);
+
+  photoObject = JSON.parse(photoObject);
+
+
+  return new Promise((resolve, reject) => {
+
+    var query = 'SELECT * from Users WHERE Users.username ="' + username + '"';
+    console.log(query);
+
+    var userId = null;
+    var photoId = null;
+
+
```

Comment:
A lot of extra whitespace. Don't bother to fix it, but keep that it mind. If you have more than 2 blank lines, that's not stylistically good,